diff -ruN reg_tmp/system.reg reg_tmp_fixed/system.reg
--- reg_tmp/system.reg	2019-10-12 14:20:02.000000000 +0800
+++ reg_tmp_fixed/system.reg	2020-09-13 22:42:20.791480251 +0800
@@ -35534,15 +35534,16 @@
 [Software\\Microsoft\\Windows\\CurrentVersion\\Winlogon] 1486365954
 #time=1d2804a40822790
 
-[Software\\Microsoft\\Windows NT\\CurrentVersion] 1511495762
-#time=1d364d825372332
-"CSDVersion"="Service Pack 3"
-"CurrentBuildNumber"="2600"
+[Software\\Microsoft\\Windows NT\\CurrentVersion] 1571657811
+#time=1d58803d4d74e20
+"CSDVersion"="Service Pack 1"
+"CurrentBuild"="7601"
+"CurrentBuildNumber"="7601"
 "CurrentType"="Uniprocessor Free"
-"CurrentVersion"="5.1"
+"CurrentVersion"="6.1"
 "InstallDate"=dword:4be5019a
 "ProductId"="12345-oem-0000001-54321"
-"ProductName"="Microsoft Windows XP"
+"ProductName"="Microsoft Windows 7"
 "RegisteredOrganization"=""
 "RegisteredOwner"=""
 "SystemRoot"="C:\\windows"
@@ -37190,7 +37191,7 @@
 
 [Software\\Tencent\\PlatForm_Type_List\\1] 1486366185
 #time=1d2804aca40187a
-"TypeName"="QQ8.9"
+"TypeName"="QQ9.4.0"
 "TypePath"="C:\\Program Files\\Tencent\\QQ\\Bin\\QQ.exe"
 
 [Software\\Tencent\\PlatForm_Type_List\\3] 1486366142
diff -ruN reg_tmp/user.reg reg_tmp_fixed/user.reg
--- reg_tmp/user.reg	2019-10-12 14:20:02.000000000 +0800
+++ reg_tmp_fixed/user.reg	2020-09-13 22:53:02.871453706 +0800
@@ -926,7 +926,7 @@
 
 [Software\\Tencent\\PlatForm_Type_List\\1] 1486366185
 #time=1d2804aca401244
-"TypeName"="QQ8.9"
+"TypeName"="QQ9.4.0"
 "TypePath"="C:\\Program Files\\Tencent\\QQ\\Bin\\QQ.exe"
 
 [Software\\Tencent\\Plugin\\VAS\\842404343] 1486366176
@@ -1053,6 +1053,7 @@
 "qqsetupex.exe"=""
 "qqsysmon.sys"=""
 "riched20"="native,builtin"
+"tencentdl.exe"=""
 "txplatform.exe"=""
 "txupd.exe"=""
 "usp10"="native,builtin"
