diff --git a/system.reg b/system.reg
index e449ec1..45f4e6e 100644
--- a/system.reg
+++ b/system.reg
@@ -38603,11 +38603,11 @@ WINE REGISTRY Version 2
 
 [Software\\Microsoft\\Windows NT\\CurrentVersion] 1600134060
 #time=1d68b0144225dc8
-"CSDVersion"="Service Pack 3"
+"CSDVersion"="Service Pack 1"
 "CurrentBuild"="7601"
-"CurrentBuildNumber"="2600"
+"CurrentBuildNumber"="7601"
 "CurrentType"="Uniprocessor Free"
-"CurrentVersion"="5.1"
+"CurrentVersion"="6.1"
 "DigitalProductId"=hex:00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,\
   00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,\
   00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,\
@@ -38617,7 +38617,7 @@ WINE REGISTRY Version 2
   00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00
 "InstallDate"=dword:4be5019a
 "ProductId"="12345-oem-0000001-54321"
-"ProductName"="Microsoft Windows XP"
+"ProductName"="Microsoft Windows 7"
 "RegisteredOrganization"=""
 "RegisteredOwner"=""
 "SystemRoot"="C:\\windows"
@@ -41069,7 +41069,7 @@ WINE REGISTRY Version 2
 
 [Software\\Tencent\\PlatForm_Type_List\\1] 1600133839
 #time=1d68b00c062f128
-"TypeName"="QQ9.3.2"
+"TypeName"="QQ9.4.1"
 "TypePath"="c:\\Program Files\\Tencent\\QQ\\Bin\\QQ.exe"
 
 [Software\\Tencent\\PlatForm_Type_List\\3] 1592536834
diff --git a/user.reg b/user.reg
index 9468a71..e31f02a 100644
--- a/user.reg
+++ b/user.reg
@@ -926,7 +926,7 @@ WINE REGISTRY Version 2
 
 [Software\\Tencent\\PlatForm_Type_List\\1] 1600133839
 #time=1d68b00c062eec6
-"TypeName"="QQ9.3.2"
+"TypeName"="QQ9.4.1"
 "TypePath"="c:\\Program Files\\Tencent\\QQ\\Bin\\QQ.exe"
 
 [Software\\Tencent\\Plugin\\VAS\\1940915894] 1603801664
